src/components/timeline/TimelineCanvas.js

```diff
@@ -178,33 +187,3 @@ export default class TimelineCanvas<HoveredItem> extends PureComponent<
     );
   }
 }
-
-/**
- * Check for shallow equality for objects, and strict equality for everything else.
- */
-function hoveredItemsAreEqual(a: any, b: any) {
```

Comment:
Because FlameChartCanvas's hitTest function returns a new object at each call, this is still needed. I'm a bit sad because this is also not very good for the GC but I don't have a better idea now -- also I don't know yet how the flame chart works so it's not a good idea to break it now ;)